R/nma.R

```diff
@@ -795,6 +817,18 @@ nma <- function(network,
       }
     }
 
+    if (".mu" %in% all.vars(regression)) {
+      if (".mu" %in% colnames(idat_agd_arm)) {
+        warn(c(
+          "Detected `.mu` in the `regression` formula and in the data.",
+          `*` = "`.mu` column in data will be ignored.",
+          i = "`.mu` is a special variable referring to the modelled baseline risk."
+        ))
+      }
+
+      idat_agd_arm$.mu <- 1L
```

Comment:
For studies without a reference treatment arm, the study intercept is not the baseline risk. So should we not set .mu to zero so that the baseline risk meta-regression does not affect such studies?

```suggestion
      ref_trt <- levels(idat_agd_arm$.trt)[1L]

      idat_agd_arm <- idat_agd_arm %>%
        dplyr::group_by(.data$.study) %>%
        dplyr::mutate(.mu = as.integer(ref_trt %in% .data$.trt)) %>%
        dplyr::ungroup()
```

Reply:
No, I don't think so - when `class_interactions` is not `"common"` the interactions with baseline risk will not cancel out, because the interactions for each non-reference treatment are different, so we need to include the BRMR terms on all arms. We use a model parameterisation where the intercept in every study is the network reference treatment ([described in this paper](https://doi.org/10.1111/rssa.12579)), so this is the right baseline risk parameter.